### PR TITLE
Vedat3.8 refactor

### PR DESCRIPTION
**Pull request by spektra2147**
…d edit category

### CODE REVIEW

File: addons/default/visiosoft/cats-module/src/Category/CategoryModel.php [hunk 1]
```diff
@@ -1,196 +1,19 @@
...
-                if ($main->parent_category_id != null) {
-                    $id = $main->parent_category_id;
-                    $z++;
-                }
-            }
-        }
-        $categories = array_reverse($categories);
-        unset($categories[count($categories) - 1]);
-        return $categories;
+        return $this->seo_description;
```
Comment (fatihalp) [inline addons/default/visiosoft/cats-module/src/Category/CategoryModel.php:16]: are you sure? @spektra2147 @diashalabi

File: addons/default/visiosoft/cats-module/src/Category/Listener/CalculatedTotalForEditedAdCategory.php [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+        return $this->categoryRepository = $categoryRepository;
+    }
+
+    public function handle(EditedAdCategory $event)
+    {
+        $ad_detail = $event->getAdDetail()->toArray();
+        $before_editing_ad = $event->getBeforeEditingParams();
+
+        //Categories New Ad
+        $category_fields_new_ad = preg_grep('/^cat/i', array_keys($ad_detail));
```
Comment (fatihalp) [inline addons/default/visiosoft/cats-module/src/Category/Listener/CalculatedTotalForEditedAdCategory.php:22]: is there any better solutions?

File: addons/default/visiosoft/cats-module/src/Category/CategoryCriteria.php [hunk 3]
```diff
@@ -19,14 +19,7 @@ public function __construct(
         $this->categoryRepository = $categoryRepository;
     }
 
-    public function getMainCats() {
```
Comment (fatihalp) [inline addons/default/visiosoft/cats-module/src/Category/CategoryCriteria.php:22]: bunu komple silelim eğer sadece çağırım yapıyorsa @spektra2147

File: addons/default/visiosoft/cats-module/src/Category/Table/CategoryTableBuilder.php [hunk 4]
```diff
@@ -86,7 +86,7 @@ class CategoryTableBuilder extends TableBuilder
      * @var array
      */
     protected $options = [
-
+        'table_view' => 'visiosoft.module.cats::table/table'
```
Comment (fatihalp) [inline addons/default/visiosoft/cats-module/src/Category/Table/CategoryTableBuilder.php:89]: it's now working with new pyro @spektra2147

File: addons/default/visiosoft/cats-module/src/CatsModulePlugin.php [hunk 5]
```diff
@@ -3,14 +3,19 @@
 use Anomaly\Streams\Platform\Addon\Plugin\Plugin;
 use Anomaly\Streams\Platform\Image\Command\MakeImageInstance;
 
-use Visiosoft\CatsModule\Category\Command\getLevel2Cats;
-use Visiosoft\CatsModule\Category\CategoryModel;
-use Visiosoft\CatsModule\Category\CategoryRepository;
+use Visiosoft\CatsModule\Category\Command\getCategoriesLevel2;
```
Comment (fatihalp) [inline addons/default/visiosoft/cats-module/src/CatsModulePlugin.php:6]: can you change it uppercase?

**Comment by fatihalp** [comment]
conflict. why we have language files here?

burada conflict ve dil dosyaları var @spektra2147

File: addons/default/visiosoft/cats-module/src/Category/Listener/CalculatedTotalForEditedAdCategory.php (hunk 2 above)
Comment (spektra2147) [reply addons/default/visiosoft/cats-module/src/Category/Listener/CalculatedTotalForEditedAdCategory.php:25]: waiting for the solutions you know. @fatihalp

File: addons/default/visiosoft/cats-module/src/Category/Table/CategoryTableBuilder.php (hunk 4 above)
Comment (spektra2147) [reply addons/default/visiosoft/cats-module/src/Category/Table/CategoryTableBuilder.php:89]: yes I'm tested

File: addons/default/visiosoft/cats-module/src/Category/CategoryModel.php (hunk 1 above)
Comment (spektra2147) [reply addons/default/visiosoft/cats-module/src/Category/CategoryModel.php:16]: yes

**Pull request merged**